Author: Nicholas Bamber <nicholas@periapt.co.uk>
Subject: Fix error in the man pages
 This is really ugly. The man pages are generated by a custom text processing
 system from filesin the doc/en/source directory. Therefore really this
 needs to be fixed in the upstream 'ej' utility. It would be better to patch
 that and submit that patch upstream.
Last-Update: 2011-02-02
Forwarded: no
--- a/doc/en/man/askmara.1
+++ b/doc/en/man/askmara.1
@@ -22,7 +22,7 @@
 
 .SH "NAME"
 .PP
-askmara - do simple dns queries
+askmara \- do simple dns queries
 .SH "DESCRIPTION"
 .PP
 .B "askmara"
@@ -35,24 +35,29 @@
 .SH "USAGE"
 .PP
 .B "askmara"
-[-n] [ -v | -t timeout] query [ server ]
+.RB [ -n ]
+.RB [ -v | -t
+.IR timeout ]
+.I query
+.RI [ server ]
 .SH "OPTIONS"
 .PP
 .TP 4
--t
-If this is present, the following argument is the askmara timeout, in
-seconds. Note that
+.BI -t
+If this is present, the following argument is the askmara
+.IR timeout ,
+in seconds. Note that
 .B "askmara"
 can not both have a user-defined
 timeout and verbose output.
 .TP 4
--v
+.BI -v
 If this is set,
 .B "askmara"
 will verbosely output the complete reply that the server sent. Note
 that this verbose output is not csv2-compatible.
 .TP 4
--n
+.BI -n
 If this is set,
 .BR "askmara" ","
 when sending out a query, will not
@@ -60,7 +65,7 @@
 remote DNS server not contact other DNS servers to answer the query in
 question.
 .TP 4
-query
+.I query
 dns record to be queried. The query has two sections:
 The type of record we desire, and the hostname we want this record for.
 .PP
@@ -102,7 +107,7 @@
 .B "Z"
 signifies that we want to ask for all RRs.
 .TP 4
-server
+.I server
 IP address of the dns server to be queried. If no server is given,
 askmara will query 127.0.0.1.
 .PP
--- a/doc/en/man/bind2csv2.1
+++ b/doc/en/man/bind2csv2.1
@@ -22,7 +22,7 @@
 
 .SH "NAME"
 .PP
-bind2csv2.py - convert zone files from BIND to MaraDNS compatible
+bind2csv2.py \- convert zone files from BIND to MaraDNS compatible
 format
 .SH "DESCRPTION"
 .PP
@@ -56,7 +56,7 @@
 zone files, and invoke the script thusly:
 
 .nf
-bind2csv2.py -c zone1 zone2 zone3
+.BI "bind2csv2.py -c" " zone1 zone2 zone3"
 .fi
 
 Substitute "zone1", "zone2", and "zone3" with a list of one or more
--- a/doc/en/man/mqhash.1
+++ b/doc/en/man/mqhash.1
@@ -22,10 +22,14 @@
 
 .SH "NAME"
 .PP
-mqhash - Simple secure password generator
+mqhash \- Simple secure password generator
 .SH "SYNOPSIS"
 .PP
-.B "mqhash [-n #] [-s] [-u] {data to hash}"
+.B mqhash
+.RB [ "-n #" ]
+.RB [ -s ]
+.RB [ -u ]
+.I {data to hash}
 .SH "DESCRIPTION"
 .PP
 .B "mqhash"
@@ -57,15 +61,17 @@
 Once the ~/.mhash_prefix file is set up, mqhash is run thusly:
 
 .nf
-mqhash -s {location}
+.BI "mqhash -s " location
 .fi
 
-Where {location} is a web site, email address, or any other text string
+Where
+.I location
+is a web site, email address, or any other text string
 that describes where a given password is located. It is recommended
-that
-one uses a consistent style for {location} so that one can remember
-passwords for web sites that one has not visited for a while.
-Mqhash does not impose a style for remembering passwords; it is up
+that one uses a consistent style for
+.I location
+so that one can remember passwords for web sites that one has not visited for
+a while. Mqhash does not impose a style for remembering passwords; it is up
 to the user to create one.
 .PP
 .B "mqhash"
--- a/doc/en/man/fetchzone.1
+++ b/doc/en/man/fetchzone.1
@@ -22,7 +22,7 @@
 
 .SH "NAME"
 .PP
-fetchzone - get dns zone from server
+fetchzone \- get dns zone from server
 .SH "DESCRIPTION"
 .PP
 .B "fetchzone"
@@ -31,7 +31,8 @@
 .SH "USAGE"
 .PP
 .B "fetchzone"
-zone_name zone_server_IP [query_class]
+.I zone_name zone_server_IP
+.RI [ query_class ]
 .SH "OPTIONS"
 .PP
 .B "zone_name"
--- a/doc/en/man/getzone.1
+++ b/doc/en/man/getzone.1
@@ -22,7 +22,7 @@
 
 .SH "NAME"
 .PP
-getzone - get dns zone from server
+getzone \- get dns zone from server
 .SH "DESCRIPTION"
 .PP
 .B "getzone"
@@ -34,19 +34,20 @@
 .SH "USAGE"
 .PP
 .B "getzone"
-zone_name zone_server_IP [query_class]
+.I zone_name zone_server_IP
+.RI [ query_class ]
 .SH "OPTIONS"
 .PP
-.B "zone_name"
-.PP
+.TP 4
+.I "zone_name"
 Name of the dns zone to be transferred.
 .PP
-.B "zone_server_IP"
-.PP
+.TP 4
+.I "zone_server_IP"
 IP address of dns server
 .PP
-.B "query_class"
-.PP
+.TP 4
+.I "query_class"
 Optional argument which can change the query class from 1 (the
 default) to 255. This may be needed for some versions of Bind.
 .SH "EXAMPLES"
--- a/doc/en/man/maradns.8
+++ b/doc/en/man/maradns.8
@@ -165,7 +165,7 @@
 cryptic error message.
 .PP
 10. After I start MaraDNS, I can not see the process
-when I run netstat -na
+when I run netstat \-na
 .PP
 11. What string library does MaraDNS use?
 .PP
@@ -432,7 +432,7 @@
 .PP
 .in -3
 \fB10. After I start MaraDNS, I can not see the process when I run
-netstat -na \fR
+netstat \-na \fR
 .PP
 Udp services do not have a prominent "LISTEN" when netstat is run.
 .PP
@@ -440,7 +440,7 @@
 like this:
 udp 0 0 127.0.0.1:53 0.0.0.0:*
 .PP
-While on the topic of netstat, if you run netstat -nap as root
+While on the topic of netstat, if you run netstat \-nap as root
 on Linux and some other *nix operating systems, you can see the names
 of
 the processes which are providing internet services.
@@ -539,7 +539,7 @@
 *
 After editing, do something like this:
 .br
-diff -u MaraDNS.c.orig MaraDNS.c > maradns.patch
+diff \-u MaraDNS.c.orig MaraDNS.c > maradns.patch
 .TP 2
 *
 Make sure the modified version compiles cleanly
@@ -794,12 +794,12 @@
 will work in MaraDNS csv2 zone files as long as the double quotes (")
 are
 replaced by single quotes (\(aq). For example, a SPF TXT record that
-looks like example.net. IN TXT "v=spf1 +mx a:colo.example.com/28 -all"
+looks like example.net. IN TXT "v=spf1 +mx a:colo.example.com/28 \-all"
 in a BIND zone file will look like
-example.net. TXT \(aqv=spf1 +mx a:colo.example.com/28 -all\(aq in a
+example.net. TXT \(aqv=spf1 +mx a:colo.example.com/28 \-all\(aq in a
 MaraDNS zone file. MaraDNS version 1.2.08 and higher can also make
 the corresponding SPF record, which will have the syntax
-example.net. SPF \(aqv=spf1 +mx a:colo.example.com/28 -all\(aq.
+example.net. SPF \(aqv=spf1 +mx a:colo.example.com/28 \-all\(aq.
 .PP
 .in -3
 \fB30. I\(aqm having problems resolving CNAMES I have set up.\fR
@@ -853,8 +853,8 @@
 .PP
 In the case of there being a NS delegation, MaraDNS handles recursive
 queries and non-recursive DNS queries differently. Basically, unless
-you use askmara with the -n option, dig with the
-+norecuse option, or nslookup with the -norec
+you use askmara with the \-n option, dig with the
++norecuse option, or nslookup with the \-norec
 option, MaraDNS will try to recursively resolve the record that is
 delegated.
 .PP
@@ -900,7 +900,7 @@
 *
 Askmara would be run thusly:
 .PP
-askmara -n Awww.joe.example.com. 10.1.2.3
+askmara \-n Awww.joe.example.com. 10.1.2.3
 .PP
 .TP 2
 *
@@ -912,7 +912,7 @@
 *
 Nslookup would be run thusly:
 .PP
-nslookup -norec www.joe.example.com 10.1.2.3
+nslookup \-norec www.joe.example.com 10.1.2.3
 .PP
 .PP
 This will allow you to see that packets MaraDNS actually sends to
@@ -1069,7 +1069,7 @@
 .TP 2
 *
 Change (or create) that line to look something like FLAGS=-O3
-In this example, MaraDNS is compiled with the -O3 option.
+In this example, MaraDNS is compiled with the \-O3 option.
 .TP 2
 *
 Note that it is important to
@@ -1080,7 +1080,7 @@
 FLAGS=-O2&nbsp;-Wall&nbsp;-DSELECT_PROBLEM and you want to compile
 at optimization level three, change this line to look like
 FLAGS=-O2&nbsp;-Wall&nbsp;-DSELECT_PROBLEM retaining the flags
-added by the configuration script. -DSELECT_PROBLEM for example,
+added by the configuration script. \-DSELECT_PROBLEM for example,
 is needed in the Linux compile or MaraDNS will have problems with
 freezing up.
 .PP
@@ -1219,7 +1219,7 @@
 .TP 2
 *
 It is necessary to have recursion disabled. This can be done either by
-compiling MaraDNS without recursive support (./configure --authonly ;
+compiling MaraDNS without recursive support (./configure \--authonly ;
 make),
 or by making sure MaraDNS does not have recursion enabled (by not
 having
--- a/doc/en/man/zoneserver.8
+++ b/doc/en/man/zoneserver.8
@@ -22,7 +22,7 @@
 
 .SH "NAME"
 .PP
-zoneserver - handle zone transfers and other TCP functions for MaraDNS
+zoneserver \- handle zone transfers and other TCP functions for MaraDNS
 .SH "DESCRIPTION"
 .PP
 .B "zoneserver"
@@ -34,12 +34,12 @@
 by default, to determine its parameters.
 .SH "USAGE"
 .PP
-.B "zoneserver -f"
-pointer_to_mararc_file
+.B zoneserver -f
+.I pointer_to_mararc_file
 .SH "OPTIONS"
 .PP
 .TP 4
--f
+.B -f
 Specifies the location of the configuration file. MaraDNS uses the same
 configuration file for both the main dns server and the zoneserver.
 .PP
--- a/doc/en/man/mararc.5
+++ b/doc/en/man/mararc.5
@@ -109,7 +109,7 @@
 initializing.
 .SH "DICTIONARY VARIABLES"
 .PP
-Here is a listing of all "dictionary"-style variables that MaraDNS
+Here is a listing of all "dictionary"\-style variables that MaraDNS
 uses:
 .PP
 .in -3
